### DISCUSSION TITLE
Can the CLI command line arguments be used when running reload programmatically?

### DISCUSSION THREAD

**Discussion by markcellus** [jprichardson/reload#320] [Q&A] [answered]
I see that the CLI command line arguments like `browser`, `hostname`, etc can be set when running reload via command line. But can these arguments also be set as options when running reload programmatically via `await reload(app)`? If not, what's the best way to set the `browser` to true so that it can open when using the `reload` function?

**Comment by alallier** [comment, marked as answer]
You cannot do this. Reload has two distinct modes.

One is in an express app where it serves the reload functionality around your existing app.

The second is as a command line tool that gives you the features you are describing.

If you want your app to auto open the browser, it's up to you to determine your use case and how you want to do that
